chia/full_node/block_height_map.py

```diff
@@ -130,9 +130,7 @@ async def create(cls, blockchain_dir: Path, db: DBWrapper2) -> "BlockHeightMap":
     def update_height(self, height: uint32, header_hash: bytes32, ses: Optional[SubEpochSummary]):
         # we're only updating the last hash. If we've reorged, we already rolled
         # back, making this the new peak
-        idx = height * 32
-        assert idx <= len(self.__height_to_hash)
-        self.__height_to_hash[idx : idx + 32] = header_hash
+        self.__set_hash(height, header_hash)
```

Comment:
see __set_hash() here: https://github.com/Chia-Network/chia-blockchain/pull/11591/files#diff-c0265f35094c3ceaa2cba46d0d4679d53b44fffe34996615de8cdb877d4cef21L211

Reply:
should we add back the assert there?

Reply:
good point. I was thinking the assert was part of `__set_hash()`, which maybe it should be